gradle.properties

```diff
@@ -16,7 +16,7 @@
 # under the License.
 group=org.apache.samza
 version=0.11.1-SNAPSHOT
-scalaVersion=2.10
+scalaVersion=2.11
```

Comment:
@craigtmc LinkedIn is still using Scala 2.10 internally. We are validating whether this change would work w/ our internal build (w/ default to 2.11). Once that is validated, we can go ahead w/ this PR. Thanks!